care/emr/resources/report/template/spec.py

```
@@ -0,0 +1,171 @@
+        for builder_key, builder_schema in schema["querysets"].items():
+            available_fields[builder_key] = {
+                field["key"] for field in builder_schema["fields"]
+            }
+
+        invalid_refs = []
+        for var in variables:
+            if var in ["loop", "current_date", "current_datetime", "current_time"]:
+                continue
+
+            parts = var.split(".")
+            if len(parts) < 2:  # noqa: PLR2004
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)